### ISSUE TITLE
Move dependencies to Maven

### ISSUE THREAD

**Issue by Kiran-Rao** [Giphy/giphy-android-sdk#131]
Hello, Jcenter is shutting down soon. The following dependencies appear to be relying on jcenter:

```
|    +--- io.alterac.blurkit:blurkit:1.1.1 FAILED
|    +--- com.facebook.fresco:fresco:2.4.0 FAILED
|    +--- com.facebook.fresco:animated-gif:2.4.0 FAILED
|    +--- com.facebook.fresco:animated-base:2.4.0 FAILED
|    +--- com.facebook.fresco:animated-drawable:2.4.0 FAILED
|    +--- com.facebook.fresco:animated-webp:2.4.0 FAILED
|    +--- com.facebook.fresco:webpsupport:2.4.0 FAILED
|    +--- com.facebook.fresco:imagepipeline-okhttp3:2.4.0 FAILED
```

**Comment by Kiran-Rao** [comment]
Related: https://github.com/facebook/fresco/issues/2603

**Comment by ALexanderLonsky** [comment]
hey @Kiran-Rao, thank you for flagging this.
We are aware of those dependencies and it is on our roadmap. 
As you mentioned, the latest Fresco version is not on Maven yet.

**Comment by oprisnik** [comment]
Hey! We've released Fresco 2.5.0 tot Maven Central, so you should be able to upgrade now.

**Comment by ALexanderLonsky** [comment]
@oprisnik thank you for the update, we'll do that soon.

**Comment by gildor** [comment]
@ALexanderLonsky Is there any chance that something will be done also with blurkit? For now, Giphy is the only library which we use which depends on jcenter dependencies

**Comment by ALexanderLonsky** [comment]
@Kiran-Rao, @gildor 
The issue must be fixed in [v2.1.1](https://github.com/Giphy/giphy-android-sdk/releases/tag/v2.1.1)

**Comment by Kiran-Rao** [comment]
Thanks @ALexanderLonsky, [v2.1.1](https://github.com/Giphy/giphy-android-sdk/releases/tag/v2.1.1) appears to have solved this.
